train/tr11-176B-ml/tr11-176B-ml.slurm

```diff
@@ -40,12 +40,10 @@ KILL_SWITCH_PATH=$MEGATRON_DEEPSPEED_REPO/kill-switch-tr11-176B-exp1
 BIGSCIENCE_REPO=/gpfswork/rech/six/uty16tp/code/big_science/bigscience
```

Comment:
We'll need to point to the "official" one.

Reply:
BIGSCIENCE_REPO=$six_ALL_CCFRWORK/code/tr11-176B-ml/bigscience